providers/caching/cdn.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const requestPromise = require('request-promise-native')
+const waterMark = 30
```

Comment:
Quick one: is the watermark and defaultIntervalMs a conf that we shall move into the fullenv.json file? or is there a particular reason to keep it hardcoded in cdn.js?

Reply:
Will do, treating it just as it happens with RATE_LIMIT_MAX (parsed as integer, no provisions for not-a-number).